tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -58,10 +58,14 @@ class CloudGov(Common):
     ###
     # Dynamic Database configuration based on cloud.gov services
     #
+    env_based_db_name = f'tdp_db_{cloudgov_space_suffix}_{cloudgov_name}'
+
+    db_name = database_creds['db_name'] if cloudgov_space_suffix == "prod" else  env_based_db_name
```

Comment:
This would use the auto-generated db name 'cw....', right?  Were you unable or didn't want to manually create that database?

Reply:
Not sure why it had to be manually created if no other env is sharing this instance?

Reply:
It doesn't have to be created, just noticed it was different than the other environments for naming convention. Not important to fix, fine as-is